CMakeLists.txt

```diff
@@ -110,6 +108,8 @@ if(CORE_SYSTEM_NAME STREQUAL osx OR
                       ${SECURITY})
 
   list(APPEND DEPLIBS ${AUDIOTOOLBOX} ${COREFOUNDATION} ${COREMEDIA} ${VIDEOTOOLBOX} ${SECURITY})
+elseif(NOT CORE_SYSTEM_NAME STREQUAL osx)
```

Comment:
Would the elseif not always be true? I.e. is this the same?

```
else()
  list(APPEND DEPLIBS ${FFMPEG_LIBRARIES})
endif()
```

Reply:
Also I think this commit and previous one can be squashed together.

Reply:
Have updated an brought together